packages/calcite-components/src/utils/text.tsx

```diff
@@ -0,0 +1,24 @@
+import { h, JsxNode } from "@arcgis/lumina";
+
+export function mark({
```

Comment:
Could we rename this to something that better conveys applying a class based on a pattern in addition to creating mark elements?

Would `highlightText` or something similar work?

Reply:
done!